session/filesync/diffcopy.go

```diff
@@ -102,6 +102,7 @@ func syncTargetDiffCopy(ds grpc.ServerStream, dest string) error {
 	if err := os.MkdirAll(dest, 0700); err != nil {
 		return errors.Wrapf(err, "failed to create synctarget dest dir %s", dest)
 	}
+	modTime := time.Now().UnixNano()
```

Comment:
I found adding various comments such as:

```suggestion
	modTime := time.Now().UnixNano() // earthly-specific
```

makes it much easier when an upstream merge produces any conflicts.